dev/NavigationView/NavigationView.xaml

```diff
@@ -346,13 +346,16 @@
                                         VerticalContentAlignment="Stretch"
                                         HorizontalContentAlignment="Stretch"
                                         Grid.Column="7" />
-
-                                    <local:NavigationViewItem
+                                    <Grid
```

Comment:
Just a thought, isn't Border more lightweight of a control compared to Grid? I'm guessing Border would work here too.

Reply:
I guess so, but on the other hand, a border (at least for me) signals a highlighting. I would not associate a Border as a "normal" container like a grid  or stackpanel.

Reply:
Ok, that makes sense it was just a thought for potential optimizing. Functionally, no problem. Thanks a lot BTW for fixing this!

Reply:
> Ok, that makes sense it was just a thought for potential optimizing. Functionally, no problem.

I will look into if using a border has the same effect regarding fixing this issue later and if that still works I will change that if that is fine with the reviewer(s) and is requested. :)



> Thanks a lot BTW for fixing this!

No problem! :)